lib/start/src/sun_flower.dart

```diff
@@ -0,0 +1,229 @@
+import 'dart:math' as math;
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:thc/start/src/za_hando.dart';
+import 'package:thc/utils/num_powers.dart';
+
+class Sunflower extends StatefulWidget {
+  Sunflower({required this.centerColor, required this.outerColor, this.bulge = 1})
+      : super(key: _key);
+
+  final Color centerColor;
+  final Color outerColor;
+
+  /// `0.0` → just a circle
+  ///
+  /// `1.0` → a bunch of semicircle bumps
+  final double bulge;
+
+  static final _key = GlobalKey<_SunflowerState>();
+  static const size = 260.0;
+  static const padding = 20.0;
+  static const petalCount = 9;
+
+  static const glow = Color(0xfffff0e0);
+  static const center = Color(0xffffc0f0);
+  static const outer = Color(0xffff55d6);
+  static const border = Color(0xffee2288);
+  static const overlayText = Color(0x50ff0080);
+
+  @override
+  State<Sunflower> createState() => _SunflowerState();
+}
+
+class _SunflowerState extends State<Sunflower> with SingleTickerProviderStateMixin {
+  late final controller = AnimationController(
+    vsync: this,
+    duration: const Duration(milliseconds: 1666),
+    upperBound: PolarPath.theta,
+  )..repeat();
+
+  @override
+  void dispose() {
+    controller.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return AnimatedBuilder(
+      animation: controller,
+      builder: (context, child) => CustomPaint(
+        size: const Size.square(Sunflower.size),
+        willChange: true,
+        painter: _SunflowerPaint(
+          centerColor: widget.centerColor,
+          outerColor: widget.outerColor,
+          bulge: widget.bulge,
+          rotation: controller.value,
+        ),
+      ),
+    );
```

Comment:
This is the first time I've implemented a `CustomPaint` widget! I'm very happy about it 😊